util/weekly_build/sites/orion.sh

```diff
@@ -26,3 +26,5 @@ UFSWM_URL="https://github.com/ufs-community/ufs-weather-model.git"
 # rt.sh parameters / arguments
 BATCHACCOUNT=epic
 RT_ARGS="-k -r"
+
+FIND_CMD="find"
```

Comment:
@RatkoVasic-NOAA @rickgrubin-noaa Please check if this is correct or if you can use an optimized version such as `lfs find` (see Blueback site config above). Thanks.